docker-compose.yaml

```diff
@@ -17,14 +17,47 @@ services:
     image: quay.io/strimzi/kafka:0.24.0-kafka-2.8.0
     command: [
       "sh", "-c",
-      "bin/kafka-server-start.sh config/server.properties --override listeners=$${KAFKA_LISTENERS} --override advertised.listeners=$${KAFKA_ADVERTISED_LISTENERS} --override zookeeper.connect=$${KAFKA_ZOOKEEPER_CONNECT}"
+      "bin/kafka-server-start.sh config/server.properties --override listeners=$${KAFKA_LISTENERS} --override advertised.listeners=$${KAFKA_ADVERTISED_LISTENERS} --override zookeeper.connect=$${KAFKA_ZOOKEEPER_CONNECT}",
+      "bin/kafka-topics.sh --bootstrap-server kafka:9092 --list",
+      "bin/kafka-topics.sh --bootstrap-server kafka:9092 --create --if-not-exists --topic orders --replication-factor 1 --partitions 1"
```

Comment:
nit to change the partitions to `4` instead of `1` so we can keep the original demo behavior
https://github.com/nodeshift-starters/reactive-koffeeshop-demo/tree/otel#how-to-run

```
docker exec -ti reactive-koffeeshop-demo_kafka_1 /bin/bash
[kafka@3a2c7aab2b4b kafka]$ cd bin
[kafka@3a2c7aab2b4b bin]$ ./kafka-topics.sh --bootstrap-server localhost:9092 --create --partitions 4 --replication-factor 1 --topic orders
Created topic orders.
exit
```
Also thanks for that automation :+1:

Reply:
okay , Thank you